packages/assistify-defaults/server/roles.js

```diff
@@ -143,59 +143,105 @@ const createConfigurationRole = function() {
+		//'change-setting-Invitation_Subject', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Invitation_Customized', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Email_Footer', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Email_Header', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Offline_Mention_All_Email', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Offline_Mention_Email', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Email', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-Custom_Translations', - revoke this permission - only admins shall be able to maintain it
 		'change-setting-Notifications_Max_Room_Members',
 		'change-setting-Mobile_Notifications_Default_Alert',
 		'change-setting-Desktop_Notifications_Default_Alert',
 		'change-setting-Audio_Notifications_Default_Alert',
 		'change-setting-Audio_Notifications_Value',
 		'change-setting-Desktop_Notifications_Duration',
-		'change-setting-UTF8_Names_Slugify',
-		'change-setting-UTF8_Names_Validation',
+		//'change-setting-UTF8_Names_Slugify', - revoke this permission - only admins shall be able to maintain it
+		//'change-setting-UTF8_Names_Validation', - revoke this permission - only admins shall be able to maintain it
 		'change-setting-Unread_Count_DM',
 		'change-setting-Unread_Count',
 		'change-setting-First_Channel_After_Login',
 		'change-setting-Favorite_Rooms',
 		'change-setting-Language',
 		'change-setting-General',
+		'change-setting-Accounts',
+		'change-setting-Accounts_ManuallyApproveNewUsers',
 		'change-setting-Accounts_BlockedUsernameList',
-		'change-setting-Accounts_BlockedDomainsList',
+		'change-setting-Accounts_RegistrationForm',
+		'change-setting-Accounts_RegistrationForm_SecretURL',
+		'change-setting-Accounts_RegistrationForm_LinkReplacementText',
+		'change-setting-Accounts_Registration_AuthenticationServices_Enabled',
+		'change-setting-Accounts_Registration_AuthenticationServices_Default_Roles',
+		//'change-setting-Accounts_BlockedDomainsList', - revoke this permission - only admins shall be able to maintain it
 		//'change-setting-Accounts_AllowedDomainsList', - revoke this permission - only admins shall be able to maintain it
-		'change-setting-Accounts_ManuallyApproveNewUsers',
-		'change-setting-Accounts_EmailVerification',
-		'change-setting-Accounts',
+		//'change-setting-Accounts_EmailVerification', - revoke this permission - only admins shall be able to maintain it
 		'change-setting-Assistify_Deactivate_request_closing_comments'
 	];
 
 	assignPermissions(CONFIGURATION_ROLE_NAME,
 		settingPermissions.concat([
 			'manage-emoji',
 			'manage-own-integrations',
-			'manage-selected-settings'
+			'manage-selected-settings',
+			'change-setting-Accounts',
```

Comment:
@ruKurz What's this permission?

Reply:
@mrsimpson as far I can see `change-setting-Accounts` does not exist. I'll remove it.

Reply:
done